internal/cmd/login.go

```diff
@@ -65,10 +64,15 @@ $ infra login infraexampleserver.com
 $ infra login --provider okta
 
 # Login with an access key
-$ infra login --key 1M4CWy9wF5.fAKeKEy5sMLH9ZZzAur0ZIjy`,
+$ export INFRA_ACCESS_KEY=1M4CWy9wF5.fAKeKEy5sMLH9ZZzAur0ZIjy
+$ infra login`,
 		Args:  MaxArgs(1),
 		Group: "Core commands:",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if server, ok := os.LookupEnv("INFRA_SERVER"); ok {
```

Comment:
This means positional args > env vars right?

Reply:
yeah